proteus/NumericalSolution.py

```diff
@@ -946,6 +1249,12 @@ def calculateSolution(self,runName):
         for index,model in enumerate(self.modelList):
             self.finalizeViewSolution(model)
             self.closeArchive(model,index)
+
+        #Destroy the mesh data structures in C because garbage collection doesnt take care of it
+        #import cmeshTools
+        #m = self.modelList[0]
+        #lm = m.levelModelList[0]
+        #cmeshTools.deleteMeshDataStructures(lm.mesh.cmesh)
```

Comment:
Removed as of https://github.com/erdc-cm/proteus/pull/450/commits/15f7670c1764fdf15eac0f639a0e2d470fe6bc37